tests/Behat/CorporateBlocksContext.php

```diff
@@ -12,6 +14,13 @@
  */
 class CorporateBlocksContext extends RawDrupalContext {
 
+  /**
+   * Site switcher configuration value.
+   *
+   * @var array
+   */
+  protected $siteSwitcherConfiguration;
```

Comment:
Let's gather the config context and use that for the backup stuff.

Reply:
Yep, done.